.github/actions/build-push-container/action.yaml

```diff
@@ -51,15 +51,11 @@ inputs:
   app_chefs_geo_address_apiurl:
     description: Proxy URL to BC Geo Address API URL
     required: true
-    default: "../api/v1/bcgeoaddress/address"
-  app_chefs_advance_geo_address_apiurl:
-    description: Proxy URL to BC Geo Address API URL for advance search
-    required: true
```

Comment:
Maybe not super clear from the diff - keeping `app_chefs_geo_address_apiurl` but using the previous value from `app_chefs_advance_geo_address_apiurl`